modules.d/90dmsquash-live/dmsquash-live-root.sh

```diff
@@ -323,12 +345,22 @@ else
     elif [ -e /run/initramfs/live/${live_dir}/ext3fs.img ]; then
         FSIMG="/run/initramfs/live/${live_dir}/ext3fs.img"
     fi
+    if [ -e /run/initramfs/live/${live_dir}/${join} ]; then
```

Comment:
Did you intend to support an uncompressed join.img?  If so, please verify that this new code path works.